deploy/cephfs/kubernetes/v1.14+/csi-nodeplugin-rbac.yaml

```diff
@@ -46,8 +48,82 @@ metadata:
 subjects:
   - kind: ServiceAccount
     name: cephfs-csi-nodeplugin
+    # replace with non-default namespace name
     namespace: default
 roleRef:
   kind: ClusterRole
   name: cephfs-csi-nodeplugin
   apiGroup: rbac.authorization.k8s.io
+
+---
```

Comment:
all policy-related changes in static templates  can go to a  separate file, isn't it? as this is only if `apiserver.enable-admission-plugins=PodSecurityPolicy` is enabled in `api-server` . if yes please update doc https://github.com/ceph/ceph-csi/tree/master/docs  when to use this file